src/main/java/mate/academy/spring/service/RoleService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.model.Role;
+
+public interface RoleService {
+    Role add(Role role);
+
+    Role getByName(Role.RoleName roleName);
```

Comment:
![image](https://user-images.githubusercontent.com/55763108/236458942-d2bc86ee-ed64-47bb-b4cb-5feda214cc97.png)
according to the requirements of the task, the input parameter must be of type String